### PR TITLE
fix(core): unexpected double-click behavior when selecting multiple options on all doc pages

### PR DESCRIPTION
**Pull request by JimmFly**
close AF-968

### CODE REVIEW

**Comment by JimmFly** [comment]
* **#8461** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8461?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @JimmFly and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/components/page-list/list.tsx [hunk 1]
```diff
@@ -61,7 +61,9 @@ const useItemSelectionStateEffect = () => {
         if (
           target.tagName === 'BUTTON' ||
           target.tagName === 'INPUT' ||
-          (e.target as HTMLElement).closest('button, input, [role="toolbar"]')
+          (e.target as HTMLElement).closest(
+            'button, input, [role="toolbar"], [data-all-doc-list="true"]'
```
Comment (pengx17) [inline packages/frontend/core/src/components/page-list/list.tsx]: I think maybe you can reuse `[data-testid=page-list-item]`

**Review by pengx17 — APPROVED** [review]

File: packages/frontend/core/src/components/page-list/list.tsx (hunk 1 above)
Comment (JimmFly) [reply packages/frontend/core/src/components/page-list/list.tsx]: I remember that data-test-id seemed to be removed in the production environment.

Comment (JimmFly) [reply packages/frontend/core/src/components/page-list/list.tsx]: I think I remembered it wrongly.

Comment (pengx17) [reply packages/frontend/core/src/components/page-list/list.tsx]: Good point. Or perhaps adding button role to the item list and then filter the event based on roles

**Review by EYHN — APPROVED** [review]

**Pull request merged**